recipe/yum_requirements.txt

```diff
@@ -0,0 +1,20 @@
+mesa-libGL-devel
+mesa-libGLU-devel
+mesa-libEGL-devel
+libX11-devel
+libXt-devel
+libXtst-devel
+libXrender-devel
+libXext-devel
+libXdmcp-devel
+libXcomposite-devel
+libXcursor-devel
+libxcb-devel
+libXau-devel
+libXi-devel
+libXScrnSaver-devel
+libXrandr-devel
+alsa-lib-devel
+xcb-util-devel
+xorg-x11-server-Xvfb
+pciutils-devel
```

Comment:
That is a lot of yum requirements, is it possible to reduce to a minimal working version or do you really need all of those?

Reply:
I probably dont need Xvfb here, but I had to use these for all my recipes that use qt